src/SFML/Main/MainWin32.cpp

```diff
@@ -41,6 +41,11 @@
 
 #include <windows.h>
 
+// Inform the Nvidia driver that this SFML application should
+// make use of the more powerful discrete GPU
+extern "C" {
+__declspec(dllexport) DWORD NvOptimusEnablement = 0x00000001;
```

Comment:
> the system power profile.

I'm sure you'll laugh but this thing on OS X has only two options (either always use discrete GPU or auto switch) and you actually need an extension to force integrated GPU. Close to useless. (It used to be an OS feature...)

So on *that* OS, it might make sense to not force using the discrete one at the application level.

Since I'm not a Windows user my perspective might not be relevant.